Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Select.hs

```diff
@@ -93,3 +107,20 @@ selectList estimateId = do
   checkIfEstimateCancelled estimate.id estimate.status
   selectedQuotes <- runInReplica $ QQuote.findAllByEstimateId estimateId
   pure $ SelectListRes $ map DQuote.makeQuoteAPIEntity selectedQuotes
+
+selectResult :: (EsqDBReplicaFlow m r) => Id DEstimate.Estimate -> m QuotesResultResponse
+selectResult estimateId = do
+  estimate <- runInReplica $ QEstimate.findById estimateId >>= fromMaybeM (EstimateDoesNotExist estimateId.getId)
+  searchRequest <- runInReplica $ QSearchRequest.findById estimate.requestId >>= fromMaybeM (EstimateDoesNotExist estimateId.getId)
+  if estimate.autoAssignEnabled
```

Comment:
As mentioned in earlier comments, we should not add `autoAssignEnabled` field inside estimate. Instead of that check if autoAssignedQuoteId has value if yes then this flow else the else case flow.